apis/v1alpha1/instrumentation_types.go

```diff
@@ -232,6 +236,30 @@ type Python struct {
 	Resources corev1.ResourceRequirements `json:"resourceRequirements,omitempty"`
 }
 
+type Ruby struct {
```

Comment:
Note: I think for the future we should probably just make a common struct called "CommonLanguageDetails" or something like that rather than needing to copy-paste this thing everytime. (not asking for that to be done with this, going to convert this comment into an issue)